### PR TITLE
Get sysInfos based on specified platform

### PR DESCRIPTION
**Pull request by Fatme**
Related to this PR https://github.com/NativeScript/nativescript-doctor/pull/30
Should be merged after this PR https://github.com/telerik/mobile-cli-lib/pull/1064

Fixes cloud builds regression problem.

### CODE REVIEW

File: lib/services/doctor-service.ts [hunk 1]
```diff
@@ -70,10 +70,8 @@ class DoctorService implements IDoctorService {
 	}
 
 	public async canExecuteLocalBuild(platform?: string): Promise<boolean> {
-		let infos = await doctor.getInfos();
-		if (platform) {
-			infos = this.filterInfosByPlatform(infos, platform);
-		}
+		let infos = await doctor.getInfos({ platform: platform });
```
Comment (rosen-vladimirov) [inline lib/services/doctor-service.ts]: You can use short syntax here:
```TypeScript
const infos = await doctor.getInfos({ platform });
```

File: lib/services/doctor-service.ts [hunk 2]
```diff
@@ -42,15 +42,15 @@ class DoctorService implements IDoctorService {
 		if (hasWarnings) {
 			this.$logger.info("There seem to be issues with your configuration.");
 			await this.promptForHelp();
+		} else {
+			this.$logger.out("No issues were detected.".bold);
```
Comment (rosen-vladimirov) [inline lib/services/doctor-service.ts]: will we print this on each `tns build ...` when there are no issues?

Comment (Fatme) [reply lib/services/doctor-service.ts]: No. This will be printed only on doctor command

**Comment by rosen-vladimirov** [comment]
run ci

**Pull request merged**